tools/asset-sync/contributing.md

```diff
@@ -0,0 +1,41 @@
+# Asset Sync Tests
+
+- Using [Ascii-flow](https://asciiflow.com/#/) to generate the art.
+- Using [pester](https://pester.dev/docs/quick-start) to test module.
+
+To properly run these tests, an internet connection is required. They are designed to work an external github repository: `Azure/azure-sdk-assets-integration`. This repository stores various test scenario branches that are duplicated as part of this pester integration test suite.
```

Comment:
I think we should define a sort of integration pipeline that runs for windows/linux/mac, and possibly some container images with different user and fs permissions.

Reply:
Added!